crates/solidity/testing/snapshots/cst_output/YulVariableDeclarationStatement/keyword_ufixed8x8/input.sol

```diff
@@ -0,0 +1 @@
+let ufixed8x8
```

Comment:
nit suggestion: adding a comment here (and similar inputs) to describe the specific corner case we are testing, to make it easier to spot mistakes in future PRs if something was accidentally changed.

```suggestion
// 'ufixed8x8' should be reserved till 0.7.1:
let ufixed8x8
```

Reply:
Done in https://github.com/NomicFoundation/slang/pull/723/commits/322f105dca48abc08f844e6fdfde296d169d17b6